test/functional/criteria_controller_test.rb

```diff
@@ -0,0 +1,133 @@
+# encoding: utf-8
```

Comment:
I know we are migrating to rspec, but I do not know if migrating these tests is a priority. I will be happy to migrate them if needed.